### PR TITLE
Zabbix sync with SnipeIT

### PR DESCRIPTION
**Pull request by Mixss**

### CODE REVIEW

File: osfv_cli/osfv_cli/osfv_cli.py [hunk 1]
```diff
@@ -5,10 +5,10 @@
 
 import pexpect
 import requests
-
-from .rte import RTE
-from .snipeit_api import SnipeIT
-from .sonoff_api import SonoffDevice
```
Comment (Mixss) [inline osfv_cli/osfv_cli/osfv_cli.py:11]: The relative imports weren't working for me, so I temporarily removed them. Should I add them back?

```
Traceback (most recent call last):
  File "~/osfv-scripts/osfv_cli/osfv_cli/./osfv_cli.py", line 8, in <module>
    from .rte import RTE
ImportError: attempted relative import with no known parent package
```

Comment (macpijan) [reply osfv_cli/osfv_cli/osfv_cli.py:11]: @Mixss Yes. How are you developing this?

With no relative modules, the packaged cannot be installed via `pip`.

For development, you should use poetry to setup virtualenv https://python-poetry.org/docs/basic-usage/

We can document this if necessary

File: osfv_cli/osfv_cli/zabbix.py [hunk 2]
```diff
@@ -0,0 +1,214 @@
+import json
+
+import requests
+import yaml
+
+
+class Zabbix:
+    CONFIG_FILENAME = "zabbix_config.yaml"
```
Comment (macpijan) [inline osfv_cli/osfv_cli/zabbix.py:8]: We have `snipeit.yml` so I would stick with `zabbix.yml` here as well to be consistent.

Comment (macpijan) [reply osfv_cli/osfv_cli/zabbix.py:8]: We should also add it to Makefile as we do with `snipeit.yml`. And document how to fill this file in README.

Comment (Mixss) [reply osfv_cli/osfv_cli/zabbix.py]: fixed in 01c86cc47707f471ecf7a197e2aae37511a9731

File: osfv_cli/osfv_cli/osfv_cli.py (hunk 1 above)
Comment (Mixss) [reply osfv_cli/osfv_cli/osfv_cli.py:11]: fixed in 2d15f24bfc4a8422873ecfbf7e3f38f4a05f3d8a , but I unfortunately didn't manage to make this work for me

Comment (miczyg1) [reply osfv_cli/osfv_cli/osfv_cli.py:11]: I have different error after building and installing:

```
Traceback (most recent call last):
  File "/home/miczyg/.local/bin/osfv_cli", line 5, in <module>
    from osfv_cli.osfv_cli import main
  File "/home/miczyg/.local/lib/python3.11/site-packages/osfv_cli/osfv_cli.py", line 10, in <module>
    from .rte import RTE
  File "/home/miczyg/.local/lib/python3.11/site-packages/osfv_cli/rte.py", line 6, in <module>
    from rtectrl_api import rtectrl
ModuleNotFoundError: No module named 'rtectrl_api'
```

Comment (macpijan) [reply osfv_cli/osfv_cli/osfv_cli.py:11]: I brought back relative imports in the rte.py as well: 18e125f4b93470136cb8981c11d55710bde8385b


As a result, I can reinstall the app and use it with no import errors:

```
make clean && make uninstall && make install
macpijan in ~/projects/github/dasharo/osfv-scripts/osfv_cli on snipeit-zabbix-sync ● ● λ osfv_cli snipeit -h
usage: osfv_cli snipeit [-h] {list_used,list_my,list_unused,list_all,list_for_zabbix,update_zabbix,check_out,user_add,user_del,check_in} ...

options:
  -h, --help            show this help message and exit

subcommands:
  {list_used,list_my,list_unused,list_all,list_for_zabbix,update_zabbix,check_out,user_add,user_del,check_in}
                        Snipe-IT subcommands
    list_used           List all already used assets
    list_my             List all my used assets
    list_unused         List all unused assets
    list_all            List all assets
    list_for_zabbix     List assets in a format suitable for Zabbix integration
    update_zabbix       Syncs Zabbix assets with SnipeIT ones
    check_out           Check out an asset by providing the Asset ID or RTE IP
    user_add            Add a new user by providing user First Name and Last Name
    user_del            Delete new user by providing user First Name and Last Name
    check_in            Check in an asset by providing the Asset ID or RTE IP

```

**Review by macpijan — APPROVED** [review]
Rebased, squashed, improved YML configuration retreiveing. Otherwise, the script looks good enough and worked for me.

**Pull request merged**